src/api/app/controllers/assignments_controller.rb

```diff
@@ -1,4 +1,5 @@
 class AssignmentsController < ApplicationController
+  before_action :require_admin, only: %i[create destroy]
```

Comment:
@danidoni I think the assignment policy should be used instead https://github.com/openSUSE/open-build-service/blob/dcf1bba9ccc2c8b7a4aca82f4b96408942950c01/src/api/app/policies/assignment_policy.rb#L5 collaborators can create/delete assignments as well

Reply:
So, looking at the [policy](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/policies/assignment_policy.rb), that uses [this](https://github.com/openSUSE/open-build-service/blob/dcf1bba9ccc2c8b7a4aca82f4b96408942950c01/src/api/app/models/assignment.rb#L30), looks like it allows to assign collaborators, that is not the same than allowing collaborators to assign someone else. I'm I right?

Reply:
@danidoni @hennevogel that should be the change that is required https://github.com/openSUSE/open-build-service/pull/18791

Reply:
I adapted the PR to use the policy now. It should be using the changes from #18791